app/src/main/java/org/simple/clinic/sync/SynceableRepository.kt

```diff
@@ -15,9 +15,9 @@ interface SynceableRepository<T, P> {
 
   fun recordsWithSyncStatus(syncStatus: SyncStatus): List<T>
 
-  fun setSyncStatus(from: SyncStatus, to: SyncStatus): Completable
+  fun setSyncStatus(from: SyncStatus, to: SyncStatus)
 
-  fun setSyncStatus(ids: List<UUID>, to: SyncStatus): Completable
+  fun setSyncStatus(ids: List<UUID>, to: SyncStatus)
```

Comment:
We should consider renaming these methods to better reflect what they are doing.